sdk/core/core-http/review/core-http.api.md

```diff
@@ -26,9 +26,9 @@ export interface AccessTokenCache {
 
 // @public
 export class AccessTokenRefresher {
```

Comment:
out of curiosity - why did we even expose this class? Are folks expected to touch it directly ever?

Reply:
Resolving since I reverted this.

Reply:
To answer the question: It was a mistake as far as we can tell :(